_blog/2019/asylo-redis-sqlite.md

```diff
@@ -0,0 +1,190 @@
+---
+title: "Enclavise Redis And SQLite By Asylo"
+overview: Technical support to run real-world applications in an enclave
+publish_date: 2019-08-09
+subtitle: Technical support to run real-world applications in an enclave
+attribution: The Asylo Team
+
+order: 18
+
+layout: blog
+type: markdown
+---
+{% include home.html %}
+
+## Asylo Enables Enclavization of Redis and SQLite
```

Comment:
With the title/subtitle, this header seems kind of out of place. Perhaps remove it or change it to something like "Summary".

Reply:
Fixed in the new commit